frontend/src/component/admin/users/UsersList/ChangePassword/ChangePassword.tsx

```diff
@@ -58,6 +60,11 @@ const ChangePassword = ({
             await changePassword(user.id, data.password);
             setData({});
             closeDialog();
+            setToastData({
+                title: 'Password changed successfully',
+                text: 'The user can now sign in using the new password.',
+                type: 'success',
+            });
```

Comment:
Pure scouting. I noticed we didn't have a toast when we changed a user's password. I think this is nicer.

Reply:
Ye that's nice!